invenio_circulation/config.py

```diff
@@ -90,6 +93,11 @@
         duration_validate=is_loan_duration_valid,
         item_available=is_item_available
     ),
+    extension=dict(
```

Comment:
For the extension policies, I noticed that the initialisation of it, happens at `utils.py`, and it is a static one. I would suggest to replace the functions with the actual values here. It is going to be cleaner, and if anyone needs to override it, they can do it in place. Either if there was some logic in the calculation of initial values I think it should be under a `config` property i.e. `CIRCULATION_EXTENSION_CONDITION`.

Reply:
See comment: https://github.com/inveniosoftware/invenio-circulation/pull/55#discussion_r207922094